src/wrappers/panel.js

```diff
@@ -0,0 +1,167 @@
+import { getContainerNode, renderTemplate } from '../lib/utils';
+import { component } from '../lib/suit.js';
+import cx from 'classnames';
+import defaultTemplates from './defaultTemplates';
+
+export default widgetFactory => optionsWithPanelOpts => {
+  const {
+    container,
+    cssClasses,
+    templates = {},
+    collapsible,
+  } = optionsWithPanelOpts;
+
+  const allTemplates = {
+    ...defaultTemplates,
+    ...templates,
+  };
+
+  const panel = renderPanel({
```

Comment:
Indeed it feels a bit weird to render the panel outside of the lifecycle. IMO it would be better to find something that match completely inside the widgets lifecycle. With the current implementation we don't touch the DOM until the user call start on the instance. This implem breaks this rule, not sure it really matters.

Reply:
Not sure either and I wasn't able to come up with a better solution since the constructor of the wrapped widget requires the container.